result.go

```diff
@@ -48,23 +48,12 @@ type RequestFilterForRunResults struct {
 // SendableResult represents a Test Case result
 // that can be created or updated via the api
 type SendableResult struct {
-	StatusID     int                `json:"status_id,omitempty"`
-	Comment      string             `json:"comment,omitempty"`
-	Version      string             `json:"version,omitempty"`
-	Elapsed      string             `json:"elapsed,omitempty"`
-	Defects      string             `json:"defects,omitempty"`
-	AssignedToID int                `json:"assignedto_id,omitempty"`
-	Checkbox     bool               `json:"custom_checkbox,omitempty"`
```

Comment:
The documentation is a bit confusing, but those do not refer custom fields, but rather to custom field *types*. For example, there is no custom field named `Checkbox` or `Milestone`, nor will there ever be one, since custom fields always start with the prefix `custom_`.

What the documentation actually means (I know this since I've tested it with a TestRail installation) is that if you define a custom field in TestRail named e.g. `CustomFoo` with the type `Checkbox`, it will be mapped to a `bool` value in the API.

For example, assume a TestRail installation has a custom field named `verified` of type "Checkbox". If I want to support it in the client, I will need to add a field as follows to `SendableResult`:

     CustomVerified bool `json:"custom_verified,omitempty"`

So, these fields as they currently exist in the code will always be empty and unused, and serve no purpose. 

My next pull request will add full support for custom fields to the client.